tests/components/pjlink/test_config_flow.py

```diff
@@ -0,0 +1,68 @@
+"""Tests for the PJLink config flow."""
+
+from unittest.mock import MagicMock, patch
+
+from pypjlink import Projector
+
+from homeassistant.components.pjlink.const import DOMAIN
+from homeassistant.config_entries import SOURCE_USER
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+from homeassistant.helpers import entity_registry
+
+NO_AUTH_RESPONSE = "PJLINK 0\r"
+
+
+def mock_projector():
```

Comment:
Please add a `mock_setup_entry` fixture in a local `conftest.py` and set pytestmark to auto-use it here in `test_config_flow.py`

```suggestion
pytestmark = pytest.mark.usefixtures("mock_setup_entry")

def mock_projector():
```